js/bundle_test.go

```diff
@@ -763,6 +763,7 @@ func TestBundleInstantiate(t *testing.T) {
 
 	t.Run("SetAndRun", func(t *testing.T) {
 		t.Parallel()
+		t.Skip("This makes no sense for a test we are basically testing that we can reset global")
```

Comment:
If that makes no sense, why should we keep it? Let's remove it?

Reply:
[42bf5d6](https://github.com/grafana/k6/pull/2571/commits/42bf5d60633874edf5180b585d63877166e3efbc)